src/services/olHelpers.js

```diff
@@ -200,7 +200,7 @@ angular.module('openlayers-directive').factory('olHelpers', function($q, $log, $
                     $log.error('[AngularJS - Openlayers] - MapBox layer requires the map id and the access token');
                     return;
                 }
-                url = 'http://api.tiles.mapbox.com/v4/' + source.mapId + '/{z}/{x}/{y}.png?access_token=' +
+                url = 'https://api.mapbox.com/styles/v1/mapbox/' + source.mapId + '/tiles/256/{z}/{x}/{y}?access_token=' +
```

Comment:
don't change this one here. 
This is the url for the the "old" [Mapbox Studio Classic, Mapbox Editor, or Mapbox map ID](https://www.mapbox.com/help/mapbox-with-openlayers/#mapbox-studio-classic-mapbox-editor-or-mapbox-map-id)